sqlmesh/migrations/v0034_add_default_catalog.py

```diff
@@ -132,9 +132,19 @@ def migrate(state_sync, default_catalog: t.Optional[str], **kwargs):  # type: ig
                 for name, snapshot_data_versions in parsed_snapshot["indirect_versions"].items()
             }
         # dbt specific migration
-        jinja_global_objs = node.get("jinja_macros", {}).get("global_objs")
-        if jinja_global_objs:
-            update_dbt_relations(jinja_global_objs, ["refs", "sources", "this"], default_catalog)
+        jinja_macros = node.get("jinja_macros")
+        if (
+            default_catalog
+            and jinja_macros
+            and "create_builtins_module" in jinja_macros
+            and jinja_macros["create_builtins_module"] == "sqlmesh.dbt"
```

Comment:
could this be one line `jinja_macros.get("create_bultins_module", "") == "sqlmesh.dbt"`?